multichain-testing/Makefile

```
@@ -0,0 +1,85 @@
+# see https://github.com/cosmology-tech/starship/blob/0e18757b8393357fc66426c5ee23da4ccf760e74/examples/getting-started/Makefile
+
+NAME = starship-getting-started
+FILE = config.yaml
+
+HELM_REPO = starship
+HELM_CHART = devnet
+HELM_VERSION = v0.2.2
+
+###############################################################################
+###                              All commands                               ###
+###############################################################################
+
+.PHONY: setup
+setup: setup-deps setup-kind
+
+.PHONY: stop
+stop: stop-forward delete
+
+.PHONY: clean
+clean: stop clean-kind
+
+###############################################################################
+###                          Dependency check                               ###
+###############################################################################
+
+.PHONY: check
+setup-deps:
+	bash $(CURDIR)/scripts/dev-setup.sh
+
+###############################################################################
+###                              Helm Charts                                ###
+###############################################################################
+
+install:
+	bash $(CURDIR)/scripts/install.sh --config $(FILE) --name $(NAME) --version $(HELM_VERSION)
+
+delete:
+	-helm delete $(NAME)
+
+###############################################################################
+###                                 Port forward                            ###
+###############################################################################
+
+.PHONY: port-forward
+port-forward:
+	bash $(CURDIR)/scripts/port-forward.sh --config=$(FILE)
+
+.PHONY: stop-forward
+stop-forward:
+	-pkill -f "port-forward"
+
+###############################################################################
+###                          Local Kind Setup                               ###
+###############################################################################
+KIND_CLUSTER=starship
+
+.PHONY: setup-kind
+setup-kind:
+	kind create cluster --name $(KIND_CLUSTER)
+
+.PHONY: clean-kind
+clean-kind:
+	kind delete cluster --name $(KIND_CLUSTER)
+
+###############################################################################
+###                          Agoric Setup                                   ###
+###############################################################################
+PROVISION_POOL_ADDR=agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346
+
+fund-provision-pool:
+	kubectl exec -i agoriclocal-genesis-0 -c validator -- agd tx bank send faucet $(PROVISION_POOL_ADDR) 1000000000uist -y -b block
+
+ADDR=agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce
+COIN=1000000000uist
+
+fund-wallet:
+	kubectl exec -i agoriclocal-genesis-0 -c validator -- agd tx bank send faucet $(ADDR) $(COIN) -y -b block
+
+provision-smart-wallet:
+	kubectl exec -i agoriclocal-genesis-0 -c validator -- agd tx swingset provision-one wallet $(ADDR) SMART_WALLET --from faucet -y -b block
```

This did not work for me because `--from` key needs to match `ADDR`.

```suggestion

KEY_NAME=user1
provision-smart-wallet:
	kubectl exec -i agoriclocal-genesis-0 -c validator -- agd tx swingset provision-one wallet $(ADDR) SMART_WALLET --from $(KEY_NAME) -y -b block
```

---

thanks - fixed